app/build.gradle

```diff
@@ -16,6 +17,10 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    packagingOptions {
```

Comment:
runs fine on my laptop without this - why is this necessary?

Reply:
The reasoning the website gave for it was "Add the packagingOptions block inside the android block to exclude the atomic functions module from the package and prevent warnings."

Reply:
Hmm what warnings do you see though? When I remove this I don't see any actual warnings

Reply:
I wasn't seeing any, I guess I'll just remove it